drivers/modem/wncm14a2a.c

```diff
@@ -265,7 +265,7 @@ static inline void _hexdump(const u8_t *packet, size_t length)
 	}
 }
 #else
-#define _hexdump(...)
+#define z_hexdump(...)
```

Comment:
Why is this z_hexdump when the renaming was from _hexdump() -> hexdump()?

Reply:
yeah, symbol is part of this driver so does not need have zephyr's namespace IMHO

Reply:
I meant `z_hexdump` doesn't exist.  `_hexdump` was renamed to `hexdump` (minus the underscore)

Reply:
I see, my comment is still valid though

Reply:
Renaming accident, I'll fix it up to just hexdump().